.vib/kube-prometheus/cypress/cypress/integration/kube_prometheus_spec.js

```diff
@@ -7,7 +7,7 @@ it('allows executing a query and displaying response data for each deployment',
   Object.keys(deployments).forEach((podName, i) => {
     const query = Object.values(deployments)[i].query;
 
-    cy.get('[role="textbox"]').clear().type(query);
+    cy.get('[role="textbox"]').clear().type(`${query}{enter}`);
```

Comment:
```suggestion
    cy.get('[role="textbox"]').clear({force: true}).type(`${query}{enter}`);
```

The force option should be here